GruntFile.js

```diff
@@ -60,10 +60,6 @@ module.exports = function (grunt) {
             search: {
                 src: ['node_modules/mustache/mustache.min.js', 'js/source/search.js'],
                 dest: 'js/dist/search.min.js'
-            },
```

Comment:
**thought**: while i don't really agree i also don't really care :)

Reply:
Gaah! Scott's rule number 72, "steve is usually right". I'll work on upgrading Uglify to handle my new code that uses fancy stuff like `const`.

Just for my own enlightenment, what part do you disagree with? Do you really think that we gain any sort of perf from minifying a 20 line file? Or maybe just because it's the right thing to do?

Reply:
For me it's the right thing to do and a trivial task that is not required often.

Reply:
When I tried running uglify after making the changes to the js, it errored out and made me think that it would be a big rabbit hole trying to get it to work. It ended up not being a big deal. This is ready for a final look. I've tested it with the new version of the function and it works.